### ISSUE TITLE
Cover page: displayed book metadata in main content area lacks labeling 

### ISSUE THREAD

**Issue by jhung** [pressbooks/pressbooks-book#174]
### Prerequisites

* [x] Can you reproduce the behavior you wish to address with [all plugins except Pressbooks deactivated](http://codex.wordpress.org/Multisite_Network_Administration#Plugins)?
* [x] Are you running the [latest version of Pressbooks](http://github.com/pressbooks/pressbooks/releases/latest/), the latest version of [Pressbooks Book](http://github.com/pressbooks/pressbooks-book/releases/latest/), the latest version of [this book theme](http://github.com/pressbooks/pressbooks-book/releases/latest/) and the [latest version of WordPress](http://codex.wordpress.org/Upgrading_WordPress)?
* [x] Are you experiencing the issue while using this book theme?
* [x] Did you [perform a cursory search](https://github.com/issues?q=+is%3Aissue+repo%3Apressbooks%2Fpressbooks-book) to see if your bug or enhancement is already listed?

### Description

In the book description in the main content area, displayed metadata lacks informative labeling. Adding proper labeling will give context for non-visual readers, and can also benefit other users.

### Steps to Reproduce

1. Load a book cover page.
2. Using a screen reader, read the contents of the page.

**Expected behavior:** 
Book title, sub-title, author, copyright, etc. should have labeling

For example: "book title: Inclusive Design Thinking. Subtitle: Why inclusion matters in everything we do. Author: Jess Mitchell. Copyright: All Rights Reserved"

**Actual behavior:**
Book title and metadata are read back as text without any context.

For example:
"Inclusive Design Thinking. Why inclusion matters in everything we do. Jess Mitchell. All Rights Reserved"

### Possible alternatives
Currently:

```HTML
<p class="book-header__author">
<span class="fn">Jess Mitchell</span>
</p>
```

Suggestion 1: Screen Reader only
```HTML
<p class="book-header__author">
<span class=”screen-reader-text”>Author:</span> <span class="fn">Jess Mitchell</span>
</p>
```

Suggestion 2: Visible and screen reader accessible
```HTML
<p class="book-header__author">
<span id="author">Author:</span> <span class="fn" aria-labelledby="author">Jess Mitchell</span>
</p>
```
